sort.php

```diff
@@ -23,6 +23,31 @@ function checkForEquality($firstNum, $secondNum)
     return $firstNum < $secondNum;
 }
 
+function quickSort($arr, $funcCallback)
+{
+    $arrSize = sizeof($arr);
+
+    if ($funcCallback($arrSize, 2)) {
```

Comment:
Why $funcCallback?

Reply:
that's the name of the function that will give as callback

Reply:
Why are you using it in this case?

Reply:
because we call the function as a callback or maybe not?

Reply:
Why do you call this function in this case?